### PR TITLE
VIR-20: Mass changes

### PR DESCRIPTION
**Pull request by Peskarhu**
* Mass changes work with the history
* Added the modal and setupping the view
* User can now choose as many as many position as needed
* The latest clicked will be open in position details view even if it
  was unopened. This works fine as it will still close if none are open
* Handle the passing of the selected positions to mass changes modal
* Fix to use local time

### CODE REVIEW

**Comment by Peskarhu** [comment]
kuva

------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------

kuva

------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------

kuva

------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------

kuva

**Comment by Peskarhu** [comment]
Whats up with those ESLint errors? I tried to fix those but was too tired today so if you have any tips

**Comment by Peskarhu** [comment]
Also made some UI smoothings and fixed the issue with modify not allowing new position names, but I will add those later:

kuva

File: backend/Virkarekisteri/src/Repositories/ChangeLogRepository.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class ChangeLogRepository(VirkarekisteriDb db) : IChangeLogRepository
 {
     public async Task<ChangeLog> AddChangeLogEntry(ChangeLog changeLogEntry)
     {
-        changeLogEntry.Timestamp = DateTime.UtcNow;
+        changeLogEntry.Timestamp = DateTime.UtcNow.ToLocalTime();
```
Comment (0x464e) [inline backend/Virkarekisteri/src/Repositories/ChangeLogRepository.cs:17]: What timezone might this "local time" be on the server which happens to run this. Not sure how it works, but I think we should explicitly specify a timezone.

**Review by 0x464e — DISMISSED** [review]

File: backend/Virkarekisteri/src/Repositories/ChangeLogRepository.cs (hunk 1 above)
Comment (0x464e) [reply backend/Virkarekisteri/src/Repositories/ChangeLogRepository.cs:17]: Ok, actually I just set a setting in azure which I believe makes the function app's time zone be our time zone.

**Comment by niinakos** [comment]
Did some testing, seems to work well! Good job. Although decision details field should be removed as what was discussed.

**Comment by Peskarhu** [comment]
Made some final modifyings: 
kuva

**Review by 0x464e — APPROVED** [review]
Good job! Works nicely

**Pull request merged**